web/source/build.sh

```diff
@@ -405,6 +420,14 @@ fi
 if [ $BUILD_UI = true ]; then
     echo Compile UI Modules...
     $compilecmd -p $NODE_SOURCE/tsconfig.ui.json
+
+    CURRENT_PATH=`pwd`
+    # Since the batch compiler for the UI modules outputs them within a subdirectory,
+    # we need to copy them up to the base /intermediate/ folder.
+    cd "$INTERMEDIATE/source"
```

Comment:
This changed due to the necessity of a `<reference path>` that _now_ ends up linking to select definitions within keyboard processor code.  Before, the highest common code path was web/source, so it compiled directly to the desired folder.